### PR TITLE
fix(init): fix init prompt installation

### PR DESCRIPTION
**Pull request by rishabh3112**
**What kind of change does this PR introduce?**

bugfix
**Did you add tests for your changes?**
N/A
**If relevant, did you update the documentation?**
N/A
**Summary**
#741

**Does this PR introduce a breaking change?**

No
**Other information**

### CODE REVIEW

**Comment by rishabh3112** [comment]
Hi @evenstensberg, `next` branch has refactored `bin` into `lib`, can you point me where to find installation prompt ?

**Comment by evenstensberg** [comment]
You can navigate the lib folder, it's not that big, and you learn the new internals

**Comment by rishabh3112** [comment]
@evenstensberg we don't do a global installation of `@webpack-cli/init` as per `next` branch. So `next` branch code doesn't encounter the error this PR supposed to solve. So there are two solutions:

1. revert global installation setup to local for `@webpack-cli/init` as in `next` branch ( I don't like this personally)
2. Get location to the global `@webpack-cli/init` package by getting `prefix` programatically using npm.

Which one I should implement?

**Comment by evenstensberg** [comment]
Do as 1. Does it and ill review

**Comment by rishabh3112** [comment]
Did that @evenstensberg

**Comment by rishabh3112** [comment]
Any review here 😅?

File: bin/prompt-command.js [hunk 1]
```diff
@@ -60,15 +59,6 @@ module.exports = function promptForInstallation(packages, ...args) {
 			options[0] = "add";
 		}
 
-		if (packages === "init") {
```
Comment (evenstensberg) [inline bin/prompt-command.js]: why is this line removed?

Comment (rishabh3112) [reply bin/prompt-command.js]: Doing local installation for `init` as decided, so this `if` is not required now in which we are replacing `-D` with global equivalents.

Comment (evenstensberg) [reply bin/prompt-command.js]: I think we are allowing global installation as well

Comment (rishabh3112) [reply bin/prompt-command.js]: Refer
https://github.com/webpack/webpack-cli/pull/742#issuecomment-460654658

Here you said for local setup.

Currently we are not allowing global installation in this scenario.

Comment (evenstensberg) [reply bin/prompt-command.js]: Oh sorry. So we need global installation. This PR needs to support that

Comment (rishabh3112) [reply bin/prompt-command.js]: Okay :+1:  will do it.

**Comment by rishabh3112** [comment]
Done @evenstensberg

**Comment by rishabh3112** [comment]
@evenstensberg NodeJS stable is failing on this PR.
Screenshot from 2019-03-09 15-39-18
On master too.
I have fixed commit messages here and is ready for review.

**Comment by rishabh3112** [comment]
@evenstensberg After investigation error is certainly due to `jest-util` package having the error as per `Node v11.11.0` (stable). it's working fine at lower node versions.

pasting error for the reference.

```
TypeError: Cannot assign to read only property 'Symbol(Symbol.toStringTag)' of object '#<process>'
      at exports.default (node_modules/jest-util/build/create_process_object.js:15:34)
```

**Comment by evenstensberg** [comment]
Any fix to this?

**Comment by rishabh3112** [comment]
@evenstenberg This is under a fix at jest here.
https://github.com/facebook/create-react-app/issues/6591

Mean while we can either use Node v11.10 or don't test on stable while it's under fix.

**Comment by rishabh3112** [comment]
Updating jest to v24.3.0 would fix this.
